src/uct/ib/dc/dc_mlx5.c

```diff
@@ -489,6 +489,26 @@ static void uct_dc_mlx5_iface_cleanup_dcis(uct_dc_mlx5_iface_t *iface)
     }
 }
 
+#if IBV_EXP_HW_TM_DC || HAVE_DC_EXP
+static uint64_t
+uct_dc_mlx5_iface_ooo_flag(uct_dc_mlx5_iface_t *iface, uint64_t flag,
+                           char *str, uint32_t qp_num)
+{
+#if HAVE_DECL_IBV_EXP_DCT_OOO_RW_DATA_PLACEMENT && HAVE_DECL_IBV_EXP_QP_OOO_RW_DATA_PLACEMENT
+    uct_ib_device_t *dev = uct_ib_iface_device(&iface->super.super.super);
+
+    if (iface->super.super.config.ooo_rw &&
+        UCX_IB_DEV_IS_OOO_SUPPORTED(dev, dc)) {
+        ucs_debug("enabling out-of-order support on %s%.0x dev %s",
```

Comment:
do we need space before QP number? and "0x" ?

Reply:
No, because:
1) 0x is passed in str when we want to print QP
2) If qp_num parameter is `0` nothing will be printed (we have %.0 specificator)

Reply:
ok